website/server/controllers/api-v3/members.js

```diff
@@ -119,6 +119,33 @@ api.getMember = {
   },
 };
 
+api.getMemberByUsername = {
+  method: 'GET',
+  url: '/members/username/:username',
+  middlewares: [],
+  async handler (req, res) {
+    req.checkParams('username', res.t('invalidReqParams')).notEmpty();
+
+    let validationErrors = req.validationErrors();
+    if (validationErrors) throw validationErrors;
+
+    let username = req.params.username.toLowerCase();
+
+    let member = await User
+      .findOne({'auth.local.lowerCaseUsername': username})
```

Comment:
are lowercase usernames unique though?

Reply:
oh, blurgh, you're right. I suppose this could be set up to return multiple...